### PR TITLE
DEV2-3414 bump version on alpha release

### PR DESCRIPTION
**Pull request by dimacodota**
DEV2-3414 bump version on alpha release

### CODE REVIEW

**Review by amircodota — APPROVED** [review]
LGTM

I suggest you write something to the entire team, explaining this change and what to expect in terms of alpha versions from now on

File: .github/workflows/alpha_release.yml [hunk 1]
```diff
@@ -4,94 +4,105 @@ on:
...
-      id: delete-prerelease
-      uses: codota/delete-tag-and-release@master
-      with:
-        delete_release: true
-        tag_name: ${{ steps.prerelease-tag.outputs.release }}
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: bump version
+        run: |
+          yarn run apply-version ${{ github.run_id }}
```
Comment (amircodota) [inline .github/workflows/alpha_release.yml:27]: I think you can use run_number which would make this number smaller and more readable

**Review by amircodota — APPROVED** [review]
LGTM, but see my comment about run_number

File: .github/workflows/alpha_release.yml (hunk 1 above)
Comment (dimacodota) [reply .github/workflows/alpha_release.yml:27]: the run_number is relative to the run an execution of the flow, no?

**Pull request merged**